kinoreel_backend/settings/development.py

```diff
@@ -51,6 +81,48 @@ def teardown_databases(self, old_config, **kwargs):
         """ Override the database teardown defined in parent class """
         pass
 
+    def run_tests(self, test_labels, extra_tests=None, **kwargs):
+        """
+        Run the unit tests for all the test labels in the provided list.
+        Labels must be of the form:
+         - app.TestClass.test_method
+        Run a single specific test method
+         - app.TestClass
+        Run all the test methods in a given class
+         - app
+        Search for doctests and unittests in the named application.
+
+        When looking for tests, the test runner will look in the models and
+        tests modules for the application.
+
+        A list of 'extra' tests may also be provided; these tests
+        will be added to the test suite.
+
+        Returns the number of tests that failed.
+        """
+        setup_test_environment()
+
+        settings.DEBUG = False
+
+        verbosity = getattr(settings, 'TEST_OUTPUT_VERBOSE', 1)
+        if isinstance(verbosity, bool):
+            verbosity = (1, 2)[verbosity]
```

Comment:
I think you can get rid of the type check if you do this:
verbosity = getattr(settings, 'TEST_OUTPUT_VERBOSE', False)
verbosity = (1, 2)[verbosity]